tests/test_fixtures/rule_based_profiler/example_notebooks/DataAssistants_Instantiation_And_Running-OnboardingAssistant-Spark.ipynb

```diff
@@ -0,0 +1,1498 @@
+    "    \n",
+    "For `expect_column_values_to_match_regex`, the values in the column be matched against a candidate list of common `regex` values which were built from the following sources: \n",
+    "\n",
+    "   - [20 Most Common Regular Expressions](https://regexland.com/most-common-regular-expressions/)\n",
+    "   - [Stackoverflow on how to test for valid uuid](https://stackoverflow.com/questions/7905929/how-to-test-valid-uuid-guid/13653180#13653180)\n",
+    "\n",
+    "* This is the regex list used by the `TextColumnsRule`.\n",
+    "\n",
+    "```python\n",
+    "CANDIDATE_REGEX: Set[str] = {\n",
+    "    r\"\\d+\",  # whole number with 1 or more digits\n",
+    "    r\"-?\\d+\",  # negative whole numbers\n",
+    "    r\"-?\\d+(?:\\.\\d*)?\",  # decimal numbers with . (period) separator\n",
+    "    r\"[A-Za-z0-9\\.,;:!?()\\\"'%\\-]+\",  # general text\n",
+    "    r\"^\\s+\",  # leading space\n",
+    "    r\"\\s+$\",  # trailing space\n",
+    "    r\"https?:\\/\\/(?:www\\.)?[-a-zA-Z0-9@:%._\\+~#=]{2,256}\\.[a-z]{2,6}\\b(?:[-a-zA-Z0-9@:%_\\+.~#()?&//=]*)\",  # Matching URL (including http(s) protocol)\n",
+    "    r\"<\\/?(?:p|a|b|img)(?: \\/)?>\",  # HTML tags\n",
+    "    r\"(?:25[0-5]|2[0-4]\\d|[01]\\d{2}|\\d{1,2})(?:.(?:25[0-5]|2[0-4]\\d|[01]\\d{2}|\\d{1,2})){3}\",  # IPv4 IP address\n",
+    "    r\"(?:[A-Fa-f0-9]){0,4}(?: ?:? ?(?:[A-Fa-f0-9]){0,4}){0,7}\",  # IPv6 IP address,\n",
+    "    r\"\\b[0-9a-fA-F]{8}\\b-[0-9a-fA-F]{4}-[0-5][0-9a-fA-F]{3}-[089ab][0-9a-fA-F]{3}-\\b[0-9a-fA-F]{12}\\b \",  # UUID\n",
+    "    }\n",
+    "```\n",
+    "\n",
+    "**Note**: The above list can be found in the [great_expectations repo here](https://github.com/great-expectations/great_expectations/blob/da2376b613843844afc041538269bb7683444b1f/great_expectations/rule_based_profiler/parameter_builder/regex_pattern_string_parameter_builder.py#L38)\n",
+    "\n",
+    "#### CategoricalColumnsRule\n",
+    "\n",
+    "The CategoricalColumnsRule will generate parameters for the following 3 Expectations:\n",
+    "\n",
+    "* `expect_column_values_to_be_in_set`\n",
+    "* `expect_column_unique_value_count_to_be_between`\n",
+    "* `expect_column_proportion_of_unique_values_to_be_between`\n",
+    "\n",
+    "\n",
+    "Categorical columns are determined to be ones that meet a certain cardinality threshold. This prevents us from calculating the number of `unique` value in a column with millions of rows, with each value being slightly different from another (which is theoretically possible). \n",
+    "\n",
+    "Great Expectations gets around this by only building the unique value set for columns that have less than a certain number of unique values, which is determined by the cardinality threshold. The default threshold is `FEW` which means Great Expectations will generate parameters for `expect_column_values_to_be_in_set()` for columns where the number of unique values are less than or equal to 100. \n",
```

Comment:
If I want to use a different value other than `FEW` how do I pass that in to the onboarding assistant? Oh I see, you describe this below. Could you provide a link to that section here?

Reply:
added